tests/pipelines/ip_adapters/test_ip_adapter_stable_diffusion.py

```diff
@@ -463,3 +472,26 @@ def test_inpainting_sdxl(self):
 
         max_diff = numpy_cosine_similarity_distance(image_slice, expected_slice)
         assert max_diff < 5e-4
+
+    def test_masks(self):
```

Comment:
Can we also add a test case for checking multiple images with masks, here?

Reply:
This test case uses two images, two masks and two ipadapters (the same ipadapter loaded twice)

Reply:
I see. Does it then make sense to separate the test cases and have more descriptive naming?

Reply:
Could we actually break this up into two cases. 

`test_ip_adapter_single_mask`
`test_ip_adapter_multiple_masks`

Just easier to tell if a specific functionality is broken.